HelpSource/Classes/Dictionary.schelp

```diff
@@ -10,6 +10,14 @@ Two keys match if they are strong::equal::. (i.e. == returns true.)
 The contents of a Dictionary are strong::unordered::. You must not depend on the order of items in a Dictionary.
 You must only rely on equality for the keys (e.g. symbols are ok, strings not). For identity matching see: link::Classes/IdentityDictionary::.
 
+Warning: code::nil:: can't be used as a Dictionary value.
```

Comment:
You could add the reason why: 
`code::nil:: can't be used as a Dictionary value because it is used to remove values`

Reply:
Also "Warning:" should probably use the `warning:: ::` construct.

Reply:
i would recommend `note:: ::`; over-using `warning` for things that aren't very surprising can wear out the meaning of the term.

Reply:
also, i think you should explain the intended behavior first -- `Setting code::nil:: as a value erases the key from the Dictionary. This means that code::nil:: itself can't be used as a Dictionary value.`